readme.md

```diff
@@ -31,6 +31,32 @@ To use the custom embed you will need to:
 <lite-youtube videoid="ogfYd705cRs"></lite-youtube>
 ```
 
+### Additional Parameters
+
+YouTube supports a variety of different [player parameters](https://developers.google.com/youtube/player_parameters).
+These may be applied by using the `params` property and attribute.
+The `autoplay` attribute defaults to `1` for the lite embed element.
+
+```html
+<!-- Example to show a video player without controls -->
+<lite-youtube videoid="ogfYd705cRs" params="&controls=0"></lite-youtube>
```

Comment:
Yea I thought about that when writing but figured it was already bloated enough. However, since this is going to be split now into multiple smaller PRs to get things in as more bite-sized chunks I can fix it with the next request.

Reply:
Got some fantastic news. Going to `URLSearchParams` solves this completely. Yay to not parsing standard defined strings manually. So this will be nicely wrapped up in the next PR with the minimal `params` addition.